src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -76,43 +85,27 @@ public void ProcessStart_UseShellExecute_OnUnix_ThrowsIfNoProgramInstalled(bool
             else
             {
                 fileToOpen = GetTestFilePath() + ".txt";
-                File.WriteAllText(fileToOpen, $"{nameof(ProcessStart_UseShellExecute_OnUnix_ThrowsIfNoProgramInstalled)}");
+                File.WriteAllText(fileToOpen, $"{nameof(ProcessStart_UseShellExecute_OnUnix_SuccessWhenProgramInstalled)}");
             }
-
-            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+            
```

Comment:
@danmosemsft VS doesnt show the whitespaces in my diff. Sorry I keep missing this.

Reply:
That's fine. You can set up VS to show whitespace, or look at git diff in the console, or set up a script or git trigger to spot extraneous spaces. Or just scan the diff in github before you press the button to make a PR.